tests/unit/specs/sails/game-state.spec.js

```diff
@@ -1,7 +1,7 @@
 import { describe, it, expect, beforeEach } from 'vitest';
-import { points } from '../fixtures/gameStates/points';
-import { resolveThree } from '../fixtures/gameStates/resolveThree';
-import { resolveNine } from '../fixtures/gameStates/resolveNine';
+import { points } from '../../fixtures/gameStates/points';
+import { resolveThree } from '../../fixtures/gameStates/resolveThree';
+import { resolveNine } from '../../fixtures/gameStates/resolveNine';
```

Comment:
Moved the sails test in to a sails directory in the unit spec folder to better isolate and match them.